### ISSUE TITLE
hint on deprecated docs / SEO

### ISSUE THREAD

**Issue by Laubeee** [Project-OSRM/project-osrm.github.com#30]
(copy of https://github.com/Project-OSRM/osrm-backend/issues/6328) 
For some reason, googling "osrm table api docs" first results leads me to [v5.5.1](http://project-osrm.org/docs/v5.5.1/api)

a hint and/or an easy way to switch api versions would be much appreciated (took me quite some time until I realized I was looking at an old doc). For inspiration, visit [an old sunpy docs page](https://docs.sunpy.org/en/v3.0.0/)

**Comment by mjjbell** [comment]
Yes the publishing is a manual process. I did the last one a while back.

It's generated from a combination of `http.md` and `libosrm.md` and doesn't appear to have anything smart like SEO support or understanding of versions.

Probably the simplest thing to do is create a new top-level 'user-docs' markdown file as an entry point, and change the link from [project-osrm.org](http://project-osrm.org/) to refer to that instead. The downside being it might be a bit harder to find the relevant docs for a specific older version.

We can do this as part of the docs tidying for the next release.

**Comment by chrisdebian** [comment]
Hi, Michael.

Do you know whether this was implemented? Can we close?

Chris

**Comment by DennisOSRM** [comment]
Docs for [latest version](https://project-osrm.org/docs/v26.4.0/index.html) were just published.
